src/java.base/windows/classes/sun/nio/fs/WindowsFileSystem.java

```diff
@@ -203,8 +203,7 @@ public Iterator<FileStore> iterator() {
     }
 
     // supported views
-    private static final Set<String> supportedFileAttributeViews = Collections
-        .unmodifiableSet(new HashSet<String>(Arrays.asList("basic", "dos", "acl", "owner", "user")));
+    private static final Set<String> supportedFileAttributeViews = Set.of("basic", "dos", "acl", "owner", "user");
```

Comment:
AFAIK, unlike regular ones immutable collections throw NPE in case one calls `Set.of().contains(null)`, so won't it be a breaking change, assuming that FileSystem could be exposed with `Path.getFileSystem()`?